### PR TITLE
Remove `SupportedStyles` from "Configuration parameters" in `.rubocop_todo.yml`

### PR DESCRIPTION
**Pull request by pocke**
Problem
===

`SupportedStyles` is not configurable, but "Configuration parameters" in .rubocop_todo.yml include it.

Solution
====

Ignore `SupportedStyles` in "Configuration parameters".

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by Drenmi** [comment]
As long as we don't name any configurable parameters with prefix `Supported`. 🙂

**Comment by bbatsov** [comment]
👍

File: lib/rubocop/formatter/disabled_config_formatter.rb [hunk 1]
```diff
@@ -105,7 +105,8 @@ def cop_config_params(default_cfg, cfg)
       end
 
       def output_cop_param_comments(params, default_cfg)
-        output.puts "# Configuration parameters: #{params.join(', ')}."
+        config_params = params.reject { |p| p.start_with?('Supported') }
```
Comment (bquorning) [inline lib/rubocop/formatter/disabled_config_formatter.rb]: I wonder, why use `start_with?` to get a partial match when you can do an exact match like this:
```rb
config_params = params - ['SupportedStyles']
```

Comment (pocke) [reply lib/rubocop/formatter/disabled_config_formatter.rb]: RuboCop has configurations that start with `Supported` except `SupportedStyles`. For example: `SupportedHashRocketStyles`

https://github.com/bbatsov/rubocop/blob/6fb5a84976567549302881de4923e21819040756/config/default.yml#L146

Comment (bquorning) [reply lib/rubocop/formatter/disabled_config_formatter.rb]: Ah

**Pull request merged**